libdebug/debugger/internal_debugger_holder.py

```diff
@@ -19,3 +28,28 @@ class InternalDebuggerHolder:
 
 
 internal_debugger_holder = InternalDebuggerHolder()
+
+
+def _cleanup_internal_debugger() -> None:
+    """Cleanup the internal debugger."""
+    for debugger in set(internal_debugger_holder.internal_debuggers.values()):
+        debugger: InternalDebugger
+
+        if debugger.instanced and debugger.kill_on_exit:
+            try:
+                debugger.interrupt()
+            except Exception as e:
+                liblog.debugger(f"Error while interrupting debuggee: {e}")
+
+            try:
+                debugger.kill()
+            except Exception as e:
+                liblog.debugger(f"Error while killing debuggee: {e}")
+
+            try:
+                debugger.terminate()
```

Comment:
If we include interrupt and kill into terminate, why we need to call them separately?